### ISSUE TITLE
Set environment variable `WT_FOLDER_PATH` to the folder path that contains `WindowsTerminal.exe`

### ISSUE THREAD

**Issue by daxgames** [microsoft/terminal#16295]
# Description of the new feature/enhancement

Set an environment variable `WT_FOLDER_PATH` to the folder path that contains `WindowsTerminal.exe`.  This variable should be set when Windows Terminal launches before the profile is launched.

This would be useful for users that run Windows Terminal in portable mode so Windows terminal profiles can be configured with paths relative to the folder wher WIndows Terminal is installed as shown below:

![image](https://github.com/microsoft/terminal/assets/7318053/f77f8e2c-7c8e-48d1-a4ce-81038c653468)

The normal user profile is configured as shown below and works perfectly because the `%CMDER_ROOT%` env variable was set dynamically before launching Windows Terminal.

![image](https://github.com/microsoft/terminal/assets/7318053/1bc3f646-e05d-4e69-8add-b8422a8cea90)

The above configuration does not work when launching as Administrator because the `%CMDER_ROOT%` env variable is not set globally on the system nor in the Administator's personal environment variables.  it produces the error below:

![image](https://github.com/microsoft/terminal/assets/7318053/ffd23d64-329c-4ae1-8e66-ebe36b563d4f)

For completeness sake the icon config being complained about in the error above is below:

![image](https://github.com/microsoft/terminal/assets/7318053/7b878169-edfc-4046-9ea6-453c91703b4a)

# Proposed technical implementation details (optional)

**Comment by zadjii-msft** [comment]
~This really falls under the same problem space as #13006 et al. We could add that, sure,~

Wait, no it doesn't. This is actually so simple. Set that variable _for terminal itself_. So that we can resolve things relative to ourself. brilliant. 

I think we were already tracking this in #15379 but I'll motion to use this as the tracking thread instead, since this has a real good idea. Thanks!

**Comment by daxgames** [comment]
The implemented solution in https://github.com/microsoft/terminal/pull/16949 does not solve this issue:

New Config:

![image](https://github.com/microsoft/terminal/assets/7318053/7e4a49ba-321a-42c5-9cae-db00a35d451e)

Upon Clicking `Save`:

![image](https://github.com/microsoft/terminal/assets/7318053/72551e75-2f50-4bda-b103-c4f8d4312695)

Launching the profile:

![image](https://github.com/microsoft/terminal/assets/7318053/c59b99e5-86ed-45fc-a7da-bbd1780b8899)

Proof the paths exist:

![image](https://github.com/microsoft/terminal/assets/7318053/5869abaa-68bc-4b52-8d53-b30752854a67)

**Comment by daxgames** [comment]
Also why can't this just be a normal env variable like the below that are set in 1.20.x:

![image](https://github.com/microsoft/terminal/assets/7318053/b0919fc3-35ef-46df-bdd3-ef2058ba9854)

Like this:

![image](https://github.com/microsoft/terminal/assets/7318053/eb99dd61-6875-43c3-beb3-08fd057a9577)

**Comment by DHowett** [comment]
![Image](https://github.com/microsoft/terminal/assets/189190/b95a9df2-e551-47c9-b778-5c1f2ab6e7e5)



It's `WT_SETTINGS_DIR` 😄

**Comment by daxgames** [comment]
See my edited comment above.

**Comment by DHowett** [comment]
Thanks!

**Comment by daxgames** [comment]
Thanks for the quick response and the fix!

I looked at the PR and it went back and forth on `%WT_SETTINGS_DIR%` and `%WT_SETTINGS_PATH%`.  After posting the issue I went to the Release Notes and saw the correct env var but it still throws the icon message.  Other than that it all works!

My only other concern is wrapping the icon path with double quotes causes it to fail.  Not sure if that would be an issue if the icon were in a path that contained spaces.
